README.md

```diff
@@ -1,19 +1,27 @@
 <h3 align="center">
-	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
-	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
-	Catppuccin Userstyles
-	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
+  <img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
+  <img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
+  Catppuccin Userstyles
+  <img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
 </h3>
 
 <h6 align="center">
-  <a href="#install">Install</a>
+  <a href="#%EF%B8%8F-install">Install</a>
+  .
+  <a href="#-contributing">Contributing</a>
   ·
   <a href="#-userstyles">Userstyles</a>
 </h6>
 
 &nbsp;
 
-## Install
+<p align="center">
+Catppuccin userstyles is a collection of userstyles for various websites and web applications that are themed with the soothing pastel colors that we all love.
```

Comment:
We should try to aim this description more like the main repository and pad it out a bit more.

1. What is a userstyle? A short one line sentence would be nice.
2. We should add the https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/palette/macchiato.png and also include the shields for star count, issues etc

Reply:
I created the sentence:
"A userstyle is a theme that applies to a given website with alterations to the css of the site."
However, this maybe a little complex.